docker/demo/Dockerfile

```diff
@@ -0,0 +1,4 @@
+FROM debian:bookworm-slim
+COPY --from=rdkitsqlite /lib/librdkit* /lib/
```

Comment:
Should this refer to the remote image we host on GHCR? I think so rather than requiring everyone to build it locally

Reply:
Hmm, possibly. I was coming at it from the angle of having the full build process be self-contained, but referring to the hosted image makes more sense for a "demo".